src/stylesheet/layouts/large_screen/_main_large.scss

```diff
@@ -262,4 +262,174 @@
+    }
+    .wrap_portfolio{
+        padding: 5%;
+        display: flex;
+        justify-content: center;
+    }
+    .porfolio{
+        display: block;
+        text-align: center;
+    }
+    .wrap_port_header{
+        text-align: center;
+        margin-bottom: 10%
+    }
+    .port_header{
+        @include header_font_large();
+    }
+    .wrap_out_imgport{
+        display: flex;
+        justify-content: center;
+    }
+    .wrap_imgport{
+        display: block;
+        justify-content: center;
+        text-align: center;
+        width: 35%;
+    }
+    .port_img1{
+        background-image: url("../../images/img1.png");
+        padding: 100%;
+        width: 100%;
+        background-size: contain;
+    }
+    .port_img2{
+        background-image: url("../../images/img2.png");
+        padding: 100% 200%;
+        width: 100%;
+        background-size: cover;
+        filter: brightness(50%);
+    }
+    .port_img3{
+        background-image: url("../../images/img3.png");
+        padding: 100%;
+        padding: 100% 200%;
+        width: 100%;
+        background-size: cover;
+    }
+    .port_img4{
+        background-image: url("../../images/img4.png");
+        padding: 100%;
+        background-size: contain;
+    }
+    .port_img5{
+        background-image: url("../../images/img5.png");
+        padding: 100%;
+        padding: 100% 200%;
+        width: 100%;
+        background-size: cover;
+    }
+    .port_img6{
+        background-image: url("../../images/img6.png");
+        padding: 100%;
+        background-size: contain;
+    }
+    .wrap_img{
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        width: 100%;
+    }
+    .text_port1{
+        position: absolute;
+        color: $color_white;
+        z-index: 1000;
+        transform: translateX(200px);
+    }
+    .text_port2{
+        text-align: left;
+        position: absolute;
+        background-color: $color_white;
+        width: 240px;
+        height: 240px;
+    }
+    .wrap_page_nav{
+        margin-top: 30px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        column-gap: 30px;
+    }
+    .nav_arr{
+        background-color: $color_yellow;
+        width: 30px;
+        height: 30px;
+        padding: 3%;
+        border-radius: 50%;
+    }
+    #left{
+        rotate: 90deg;
+    }
+    #right{
+        rotate: -90deg;
+    }
+    .choose_nav{
+        @include font_large();
+        color: $color_black;
+        transition: 0.25s;
+        width: 20px;
+        height: 20px;
+        border-radius: 50%;
+    }
+    .choose_nav:hover{
+        background-color: $color_yellow;
+    }
+    .text_port1{
+        text-align: left;
+    }
+    .header_img{
+        @include header_font_medium();
+        margin-bottom: 10px
+    }
+    .time_img{
+        @include font_large();
+        margin-bottom: 20px;
+    }
+    .scr_img{
+        width: 230px;
```

Comment:
dont set w h for img

Reply:
This is scriprt of image and it is p tag, not an imagr 🤧

Reply:
same